### ISSUE TITLE
Error: list index out of range in predictor.py

### ISSUE THREAD

**Issue by colingoldberg** [ClimbsRocks/auto_ml#95]
Hi,

I got error 'list index out of range' running test_script.py on numerai_training_data (9/29/2016). I made only one change to test_script.py - to refer to the numerai file I downloaded.

$python test_script.py full
Welcome to auto_ml! We're about to go through and make sense of your data using machine learning
Successfully performed basic preparations and y-value cleaning
Created estimator_names and scoring

********************************************************************************************
About to fit the pipeline for the model RidgeClassifier to predict target
Started at:
2016-09-29 16:19:30
Finished training the pipeline!
Total training time:
0:00:13

Here are the results from our RidgeClassifier
Traceback (most recent call last):
  File "test_script.py", line 54, in <module>
    ml_predictor.train(training_data, X_test=X_test, y_test=y_test)
  File "/usr/local/share/auto_ml/auto_ml/predictor.py", line 452, in train
    self.perform_grid_search_by_model_names(estimator_names, scoring, X, y)
  File "/usr/local/share/auto_ml/auto_ml/predictor.py", line 570, in perform_grid_search_by_model_names
    self._print_ml_analytics_results_regression()
  File "/usr/local/share/auto_ml/auto_ml/predictor.py", line 675, in _print_ml_analytics_results_regression
    potential_impact = feature_ranges[col_idx] * trained_coefficients[col_idx]
IndexError: list index out of range

Looking forward to the fix!

Colin Goldberg

**Comment by ClimbsRocks** [comment]
Thanks for reporting, Colin! It looks like there's some bug with how we're working with Ridge. 

I'll look into that today and should have it fixed up shortly. Thanks again for filing the issue!

**Comment by ClimbsRocks** [comment]
cool- just fixed that bug. it's actually a bit of legacy code i've been meaning to remove. At one point, before we feature scaled by default, we calculated the reasonable range for each feature, and used that to calculate our own version of a normalized number for each coefficient. 

Now, since we feature scale by default, that code's no longer necessary, so I haven't bothered updating it for a few of the recent changes. Thanks for reporting this! Should be fixed in the commit and release I just pushed out, v1.2.0.
